README.md

```diff
@@ -1,6 +1,9 @@
 # RHOAS CLI
+<p align="center">
+  <img alt="Logo" src="https://user-images.githubusercontent.com/11743717/127519981-97c76ae4-f17b-4ac8-8b4d-365bfa4a6374.png">
+</p>
 
-Repository for the RHOAS CLI.
+`rhoas` is a command-line client for managing all of your managed services on Kubernetes.
```

Comment:
Not sure about this. I want to be more descriptive than "Repository for the RHOAS CLI." while also remaining neutral in terms of upstream versus product language.

Reply:
`managed` was replaced by `application`